### DISCUSSION TITLE
Is it possible to make deepgram actually print the text for eem and eeh, mmm ...

### DISCUSSION THREAD

**Discussion by haikyuu** [deepgram/community#110] [General help] [answered]
I'm exploring deepgram for transcribing audio, and I plan on using the transcriptions to assist in editing the actual media. So I need it to actually detect the filler words and print them as part of the transcriptions

Is it possible with deepgram?

**Comment by shirgoldbird** [comment, marked as answer]
Hi @haikyuu! We're actively working on adding this capability to our product. If you're interested in beta testing, let me know :) 

A couple questions...

1. What filler words would you like to see transcribed?
2. Which models would you like to be able to transcribe filler words?

**Reply by haikyuu** [reply]
Yes, I'm interested in beta testing :) 
1. All of them? Or the most popular ones? I don't have control over what our users/teachers say (https://scrimba.com) ... But I guess, they'd be thinking (eeh, mmm ...) or surprised (ooh, aah, aha ...)
2. I need very precise word transcripts, so the enhanced model was better in my experience

**Reply by shirgoldbird** [reply]
@haikyuu Thanks for the feedback! I see you also reached out via our support portal. I've added you to our list of customers for this beta and will be in touch via email with more details next month.

**Reply by dceddia** [reply]
@shirgoldbird I'm also very interested in this feature and would love to get in on the beta testing if possible! :) I think we've got similar goals of editing videos using the transcription.

I'm thinking of words like um, uh, ah, ehh, hm, mmm. I've been able to get Nova and the default model to find `um`, `uh`, and `ah` using the `keywords` param, but it seems to miss many of them. I tried a few boost settings. `2` and `3` were pretty similar (it found some, instead of none), and at `10` it started to go overboard by splitting up some of the real words into filler words 😄

A few more I'm thinking of looking for are things like "so", "sort of", "you know", "I mean", etc. I've only just started playing around with Deepgram and I'm guessing it'll already find these pretty well though since they're normal words.

**Reply by shirgoldbird** [reply]
@dceddia @haikyuu Great news! We have a model ready for beta testing. If you can send me your Deepgram project IDs (which can be found in our Console), I can get you access in exchange for your feedback. :)

You can send those to me by email—it's my first name and my last name (Shir Goldberg) separated by a period, at deepgram.com.

**Reply by ramild** [reply]
@shirgoldbird Hey! We'd love this feature too. Could you add me to the beta test?
And does the beta work with Nova and Whisper?

**Reply by shirgoldbird** [reply]
Hi @ramild! We're no longer accepting participants to the beta, as we're very close to a full production release. We plan to announce this feature within the next month.

When we release this feature, it will initially only work with our Nova general model. We have no plans to add it to Whisper, but do plan to extend it to other Deepgram models and tiers, such as Nova phonecall or Enhanced general.

**Comment by swahnfrayeLC** [comment]
We would also be very interested in this feature, detecting filler words is a core part of our product. We are currently using the OpenAI Whisper API to detect these. But would switch to DeepGram Nova if it could detect filler words for us.
